### PR TITLE
Github Parsers and Init Container

### PR DESCRIPTION
**Pull request by Loakeshvaran-SK-ML**

### CODE REVIEW

File: consumer.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+		data := bodyData["body"]
+		log.Println(header, data)
+		Parser(header.(string), data.(string))
+	}
+
+	return nil
+}
+
+func main() {
+    conn, err := amqp.Dial("amqp://guest:guest@localhost:5672/")
```
Comment (Dannana-Venkataramana-ML) [inline consumer.go:53]: Read username, password, URL and port number from config map

File: consumer.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
```
Comment (Dannana-Venkataramana-ML) [inline consumer.go:6]: Use this github.com/sirupsen/logrus this package for logging

**Review by Dannana-Venkataramana-ML — COMMENTED** [review]
Please provide proper message for pull request

**Comment by Dannana-Venkataramana-ML** [comment]
Where is the folder structure

File: consumer.go (hunk 2 above)
Comment (Loakeshvaran-SK-ML) [reply consumer.go]: Added the logrus

**Review by Dannana-Venkataramana-ML — APPROVED** [review]
LGMT

**Pull request merged**